adminutils/adminutils.py

```diff
@@ -234,6 +234,51 @@ async def emote_steal(self, ctx, name: str, message_id: discord.Message, *roles:
         except discord.HTTPException as e:
             await ctx.send(chat.error(_("An error occurred on adding an emoji: {}").format(e)))
 
+    @emoji.command(name="member", aliases=["user"])
+    async def member_steal(self, ctx, name: str, member: discord.Member, *roles: discord.Role):
+        """
+        Add an emoji from a specified member's status
+        Use double quotes if role name has spaces
+
+        Examples:
+            `[p]emoji member Example @PhenoM4n4n`
+            `[p]emoji message RoleBased 462364255128256513 EmojiRole`
+        """
+        emoji = None
+        if member.activity:
+            if member.activity.emoji:
+                if member.activity.emoji.is_custom_emoji():
```

Comment:
["Are you Egyptian? What's with these fucking scope pyramids!?"](https://github.com/FPtje/GLuaFixer/blob/841cc43a767439abbec65e6136a303a2f15aefc1/src/GLuaFixer/AG/ASTLint.ag#L253)

This can be just a one line, instead of 3 ifs

```suggestion
        if member.activity and member.activity.emoji and  member.activity.emoji.is_custom_emoji():
```